torch_xla/core/xla_builder.py

```diff
@@ -1016,9 +1017,12 @@ def call_jax(jax_func,
   works. If you get tracing overhead, check if `jax_func` is being redefined all the time.
   A common mistake is defining `jax_func` as a local function, e.g. during a training step.
   """
-  import jax
   kwargs = kwargs or {}
-  flattened, _spec = jax.tree.flatten((args, kwargs))
+  flattened, _spec = tree_flatten((args, kwargs))
```

Comment:
I think the `tree_flatten` also came from an old version of the code. It failed a test because the PyTorch interpretation of PyTree is different from JAX wrt None values.

Reply:
done.